content/en/xtc/xtc-release-notes/v99.md

```diff
@@ -0,0 +1,41 @@
+---
+title: XTC v99
+linkTitle: XTC v99
+
+weight: 117
+type: docs
+
+date: 2025-05-12
+
+description: 2025-05-13
+
+sitemap:
+  changefreq: never
+  priority: 0.1
+---
+
+## Load Testing
+
+* Whenever a load test changes state, users can be notified, currently only via Slack. Since not all users have Slack, we have started to include email as well. As a first step, you can now configure a list of custom email addresses at a load test, and this list will be notified when the load test finishes. See the new *Notifications* tab on the details page of a load test.
+* The REST API for load testing projects has been enhanced to allow the configuration of custom machines for a load test. This includes:
+    * The list of custom machines (specified as `<host-or-ip>:<port>`).
+    * The password required to communicate with these machines.
+    * The number of XLT agent processes per custom machine.
+    * Whether to remove regular cloud machines from the load test.
+
+  For full details, see the "update" variant of the [action endpoint](https://xtc.xceptance.com/exploreApi#post-/public/api/v2/orgs/-org-/projects/-project-/load-tests/-id-).  
+* Previously, XTC used a mirror of Maven Central provided by Google when resolving dependencies for your load testing projects. As this mirror seems to be no longer maintained, XTC now points directly to Maven Central.
+* Currently available XLT execution environments:
+    * XLT 8.x → [8.6.0]({{< relref "/xlt/release-notes/8_6_x" >}}) (deprecated, will be **retired in July, 2025**)
+    * XLT 9.x → [9.0.0]({{< relref "/xlt/release-notes/9_0_x" >}}) (**[migration needed]({{< relref "/xlt/release-notes/9_0_x#test-suite-migration" >}})**)
+
+
+## Monitoring
+
+* On the *Dashboard* and the *History* pages, the colors used to represent the different states of a scenario execution have been standardized across charts, tiles, and icons.
```

Comment:
Just to avoid the impression that we created a standard.
```suggestion
* On the *Dashboard* and the *History* pages, the colors used to represent the different states of a scenario execution have been made consistent across charts, tiles, and icons.
```